troupon/static/scss/partials/_resets.scss

```diff
@@ -0,0 +1,59 @@
+
+/* ==========================================================================
+Resets (overrides for Bootstrap and browser-defaults)
+========================================================================== */
+
+body {
+	appearance: none;
+}
+
+h2, h3, h4 {
+	margin: 0px 0px 10px 0px;
+}
+
+ul, ol {
+	padding: 0px;
+	margin: 0px;
+}
+
+.row {
+	margin: 0px;
```

Comment:
Line should be indented with spaces, not tabs<br>`0px` should be written without units as `0`